kpi/deployment_backends/kobocat_backend.py

```diff
@@ -863,3 +958,48 @@ def __prepare_as_drf_response_signature(requests_response):
 
         return prepared_drf_response
 
+    @staticmethod
+    def __validate_payload(payload: dict) -> dict:
+        """
+        Validating the request payload for bulk updating of submissions
+        """
+        if not 'submission_ids' in payload:
+            raise KobocatBulkUpdateSubmissionsException(
+                detail=_('`submission_ids` must be included in the payload')
+            )
+
+        if not isinstance(payload['submission_ids'], list):
+            raise KobocatBulkUpdateSubmissionsException(
+                detail=_('`submission_ids` must be a list')
```

Comment:
i haven't looked to see if this is actually the case, but if this is a message intended for a client, and the client sent JSON, then we should call this an "array" to avoid confusion

Reply:
Good point 💯 it was intended for the client, so I've updated it with 'array'